NorthernWinterBeat/NorthernWinterBeat/Pages/Shared/_Layout_Index.cshtml

```diff
@@ -43,6 +43,24 @@
     <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
     <script src="~/js/site.js" asp-append-version="true"></script>
 
-    @RenderSection("Scripts", required: false)
+
+
+        <script>
+            $(document).ready(function () {
+                $("#inForm1").submit(function () {
+                    $("#btnSubmit1").attr("disabled", true);
+                    return true;
+                });
+            });
+        </script>
+        <script>
+            $(document).ready(function () {
+                $("#inForm2").submit(function () {
+                    $("#btnSubmit2").attr("disabled", true);
+                    return true;
+                });
+            });
+        </script>
```

Comment:
Du må gerne lige prøve at se, om du har mulighed for at generalisere disse ned til én enkelt.
Måske skal det ikke være ID'er (#inform?) men en class man sætter på formen. Og så er der muligvis en metode i JQuery til at finde knappen ud fra en class inde i det specifikke form. Den form der submittes, kan i Callback funktionen nok findes ved at skrive $(this)